ddtrace/context.py

```diff
@@ -94,6 +94,15 @@ def sampling_priority(self, value):
                 return
             self._metrics[SAMPLING_PRIORITY_KEY] = value
 
+    @property
+    def traceparent(self):
```

Comment:
Unless we intend to make this public, we can add it to the internal interface.
```suggestion
    def _traceparent(self):
```